etl/steps/data/garden/health/2022-12-28/karlinsky.meta.yml

```diff
@@ -0,0 +1,50 @@
+dataset:
+  sources:
+  #   -
+  #     name: Maddison Project Database 2020 (Bolt and van Zanden, 2020)
+  #     published_by: Bolt, Jutta and Jan Luiten van Zanden (2020), “Maddison style estimates of the evolution of the world economy. A new 2020 update“.
+  #     publisher_source: |
+  #       The Maddison Project Database is based on the work of many researchers that have produced estimates of...
+
+tables:
+  deaths_karlinsky:
+    title: International Completeness of Death Registration 2015-2019 – Karlinsky (2021)
```

Comment:
Is it necessary to include the date range in the title?

Reply:
Yes, that's the title of the dataset and paper:

- repository: https://github.com/akarlinsky/death_registration
- paper: https://www.medrxiv.org/content/10.1101/2021.08.12.21261978v2